daemon/algod/api/algod.oas2.json

```diff
@@ -3321,6 +3325,49 @@
         }
       }
     },
+    "SimulateTransactionGroupResult": {
+      "description": "Simulation result for an atomic transaction group",
+      "type": "object",
+      "required": [
+        "txn-results"
+      ],
+      "properties": {
+        "txn-results": {
+          "description": "Simulation result for individual transactions",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SimulateTransactionResult"
+          }
+        },
+        "failure-message": {
+          "description": "If present, indicates that the transaction group failed and specifies why that happened",
+          "type": "string"
+        },
+        "failed-at": {
+          "description": "If present, indicates which transaction in this group caused the failure",
+          "type": "array",
```

Comment:
Why is this an array?  The description appears to say it will be a single integer.

Reply:
It's an array because we show you the path to the failing txn. E.g. if the second txn is an app call whose third inner failed, `failed-at` would be `[1,2]`

Reply:
Can we explain that succinctly(?) in the description?  Perhaps it's too much.

Reply:
I made an attempt in 9557a66b6, let me know what you think

Reply:
I think including that in the description is nice - the endpoint only returns the first error so it could be ambiguous whether `failed-at` is reporting every error in the group, or the path to the failed transaction.